fideslib/db/base_class.py

```diff
@@ -143,7 +143,7 @@ def all(cls: Type[T], db: Session) -> list[T]:
         return db.query(cls).all()
 
     @classmethod
-    def filter(cls: Type[T], db: Session, *, conditions: list[Any]) -> Query:
+    def filter(cls: Type[T], db: Session, *, conditions: Any) -> Query:
```

Comment:
I'm just seeing two types in Fidesops: 
- a [SQLAlchemyBinaryExpression ](https://docs.sqlalchemy.org/en/14/core/sqlelement.html#sqlalchemy.sql.expression.BinaryExpression)
  -  Examples: `((Rule.key == 4 and Rule.policy_id == 5)`,  
  - `(AuthenticationRequest.connection_key == data["connection_key"])` 
- SQLAlchemy BooleanClauseList: 
   - Examples: ` ((webhook_cls.policy_id == policy.id) & (webhook_cls.key == webhook_key))`


Maybe if we change that latter `&` to an `and` in fidesops, then we could turn every fidesops usage into a BinaryExpression.

Reply:
If we know that is all there will be I can set it to take either of those 2. What do you think about that?

Reply:
that works too!